assets/blocks/course-list-block/featured-label/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEntityProp } from '@wordpress/core-data';
+
+const FeaturedLabel = ( { postId, children } ) => {
```

Comment:
This could use a JSDoc comment. [Example](https://github.com/Automattic/sensei/blob/trunk/assets/blocks/learner-courses-block/learner-courses-edit.js#L78).

Reply:
a55789205be6bc46cda22acd97373492942a18f4